modules/uncertainty/R/get.parameter.samples.R

```diff
@@ -57,16 +57,20 @@ get.parameter.samples <- function(settings,
     ## Load posteriors
     if (!is.na(posterior.files[i])) {
       # Load specified file
-      load(posterior.files[i])
+      env <- new.env()
+      load(posterior.files[i], envir = env)
+      post.distns <- env$post.distns
+      prior.distns <- env$prior.distns
```

Comment:
@moki1202 I think this changes some assumptions in the code below: The calls to `exists` below (line 64, 113, 118) are checking whether variables named `post.distns` and `prior.distns` are defined at all, but after this assignment they'll always exist and will be NULL if they weren't present in the loaded `posterior.file`.

The obvious fixes I see are either:
* define the variables conditionally (`if (hasName(env, "post.distns")) post.distns <- env$post.distns `)
* Keep the assignments as they are and replace all `exists(x)` with `!is.null(x)`.
I think I prefer the second option -- it makes it obvious what objects we're expecting the file to contain, while `exists` is hard to reason about and has bitten us in this same way before.